pkg/master/master.go

```diff
@@ -212,11 +216,14 @@ type Master struct {
 	// External host is the name that should be used in external (public internet) URLs for this master
 	externalHost string
 	// clusterIP is the IP address of the master within the cluster.
-	clusterIP            net.IP
-	publicReadWritePort  int
-	serviceReadWriteIP   net.IP
-	serviceReadWritePort int
-	masterServices       *util.Runner
+	clusterIP           net.IP
+	publicReadWritePort int
+	serviceReadWriteIP  net.IP
+	// Ports to be exposed for the master service
```

Comment:
This explains WHAT but not WHY.

Sorry to be a pain - this is the sort of thing that i would un-do if I saw it without a good justification